### PR TITLE
Count letters - vowels and consonants, fixes #24

### LINKED ISSUE #24: Count letters: vowels and consonants [P2]

### PR DESCRIPTION
**Pull request by KrzysztofRozankiewicz**
Fixes issue #24

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -30,24 +30,30 @@ def print_menu():
...
+
+                print(" Ilość samogłosek w pliku ",filename," to ", str(letters))
+                print(" Ilość spółgłosek w pliku ",filename," to ", str(letters2))
+        except FileNotFoundError:
+                print(" ** Brak pliku ",filename, " **")
+        except Exception:
+                print(" ** Nie mogę otworzyć pliku ",filename)
 
-		
+
```
Comment (Ellectronx) [inline main.py:56]: Why spaces?

**Review by TraitPHP — COMMENTED** [review]
50 & 56 line

File: main.py (hunk 1 above)
Comment (KrzysztofRozankiewicz) [reply main.py:56]: Why not?

Comment (mzylowski) [reply main.py:56]: eh...
https://softwareengineering.stackexchange.com/questions/57/tabs-versus-spaces-what-is-the-proper-indentation-character-for-everything-in-e

**Pull request merged**